src/components/Details/DetailsContent.js

```diff
@@ -0,0 +1,99 @@
+import { useTimeZoneConversion } from '@dhis2/app-runtime'
+import i18n from '@dhis2/d2-i18n'
+import { IconChevronDown24, IconChevronUp24 } from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React, { useState } from 'react'
+import { selectedLocale } from '../../utils'
+import { StatusIcon } from '../List'
+import { Content } from './Content'
+import css from './Details.module.css'
+
+const CompletedTime = ({ finishedTime, latest }) => {
+    const { fromServerDate } = useTimeZoneConversion()
+    const latestRun = fromServerDate(finishedTime)
+
+    const formattedLatestRun = Intl.DateTimeFormat([selectedLocale], {
+        dateStyle: 'short',
+        timeStyle: 'short',
+    }).format(latestRun)
+
+    return (
+        <header title={latestRun.getClientZonedISOString()}>
+            {latest
+                ? i18n.t('Latest error {{time}}', {
+                      time: formattedLatestRun,
+                      interpolation: { escapeValue: false },
+                  })
+                : i18n.t('Error {{time}}', {
+                      time: formattedLatestRun,
+                      interpolation: { escapeValue: false },
+                  })}
+            <StatusIcon count={0} />
+        </header>
+    )
+}
+
+CompletedTime.propTypes = {
+    finishedTime: PropTypes.string,
+    latest: PropTypes.bool,
+}
+
+const ExpandableContent = ({ latest, artifact, item }) => {
```

Comment:
I don't think this have to know about "artifact". Eg. this could just take `children`. And conditionally render `children` based on `expanded`. Then `DetailsContent` will just wrap `Content` with this component.